components/tag-list/tag-list-item-mixin.js

```diff
@@ -209,12 +215,11 @@ export const TagListItemMixin = superclass => class extends LocalizeCoreElement(
 		`;
 	}
 
-	_renderTag(tagContent, options) {
-		if (!options) options = {};
+	_renderTag(tagContent, options = {}) {
+		if (options.labelText?.constructor !== String) throw new TypeError('options.labelText must be a string');
```

Comment:
Oh, I guess we'll want to have the `users` PR ready to go to merge right after this? Or the user tag items will start throwing?

Reply:
Oh yikes! Yep.